README.md

```diff
@@ -27,23 +29,225 @@ You can use the package as follows:
+
+The connection to Google Cloud Platform has been done using the [google-cloud-core](https://googleapis.dev/python/google-api-core/latest/index.html) package.
+
+We provide a handler that allows doing queries on BigQuery services.
+
+The use of this subpackage requires having Google Cloud Platform credentials, these can be passed as arguments in the initializer of the handler class directly, or you can store them in AWS Secrets Manager and retrieve them during the initialization using the `from_aws_secrets_manager` method instead.
+
+##### Examples of initialization
+
+```python
+from santoku.google import BigQueryHandler
+
+bigquery_handler = BigQueryHandler(
+    type="your_type",
+    project_id="your_project_id"
+    private_key_id="your_private_key_id"
+    private_key="your_private_key"
+    client_email="your_client_email"
+    client_id="your_client_id"
+    auth_uri="your_auth_uri"
+    token_uri="your_token_uri"
+    auth_provider_x509_cert_url="your_auth_provider_x509_cert_url"
+    client_x509_cert_url="your_client_x509_cert_url"
+)
+```
+or
+```python
+bigquery_handler = BigQueryHandler.from_aws_secrets_manager(
+    secret_name="your_secret_name"
+)
+```
+
+##### Example of queries
+```python
+query_results = bigquery_handler.get_query_results(query="SELECT * FROM `your_table`")
+```
+
+### Salesforce
+
+Salesforce is a Customer Relationship Management (CRM) platform that gives to the marketing, sales, commerce, and service depertments a single, shared view of every customer.
+
+The connection to Salesforce has been done using the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/quickstart.htm).
+
+The use of this subpackage requires having Salesforce credentials, these can be passed as arguments in the initializer of the handler class directly, or you can store them in AWS Secrets Manager and retrieve them during the initialization using the `from_aws_secrets_manager` method instead.
+
+This subpackage provide methods to insert/modify/delete salesforce object records. You can perform operations by doing HTTP requests directly or using methods with higher level of abstraction easier to handle, the lasts ones are just wrappers of the HTTP request method. To obtain records you can perform queries using SOQL.
+
+The unit tests requires valid Salesforce credentials to be executed. The tests are implemented in the way that no new data will remain in the account and no existent data will be modified. However, having Salesforce credentials for sandbox use is recommended.
+
+##### Example of initialization
+
+```python
+from santoku.salesforce import ObjectsHandler
+
+objects_handler = ObjectsHandler(
+    auth_url="your_auth_url",
+    username="your_username",
+    password="your_password",
+    client_id="your_client_id",
+    client_secret="your_client_secret",
+)
+```
+or
+```python

```

Comment:
Since we already talked about the alternative secrets manager class method, simply make this a single snippet with one init and the examples one after the other.